pyanaconda/network.py

```diff
@@ -686,7 +686,7 @@ def ksdata_from_ifcfg(devname, uuid=None):
     if devname in nm.nm_devices():
         # virtual devices (bond, vlan, ...) not activated in installer
         # are not created so guard these checks
-        if nm.nm_device_is_slave(devname):
+        if nm.nm_device_is_slave(devname) and nm.nm_device_type_is_ethernet(devname):
```

Comment:
The comment actually talks about the guard above (if devname in nm.nm_devices()), the changed line is just one of "these checks". (The change is that we want to ingore slaves only if they are ethernet devices, ie not bond devices which - unlike ethernet slaves - should have its own kickstart network command)
Anyway, the comment should be moved above the guard line so it is clear what it is talking about?